google-cloud-bigquery/lib/google/cloud/bigquery/standard_sql.rb

```diff
@@ -273,6 +273,17 @@ def numeric?
             type_kind == "NUMERIC".freeze
           end
 
+          ##
+          # Checks if the {#type_kind} of the field is `BIGNUMERIC`.
+          #
+          # @return [Boolean] `true` when `BIGNUMERIC`, `false` otherwise.
+          #
+          # @!group Helpers
+          #
+          def bignumeric?
+            type_kind == "BIGNUMERIC".freeze
```

Comment:
Why freeze?

Reply:
Maybe Rubocop demanded it when [this class was originally added in July 2019](https://github.com/googleapis/google-cloud-ruby/commit/0a831240099308fcded4dc282864d691191b1572)? I was just sticking to the existing pattern.